README.md

```diff
@@ -159,6 +160,16 @@ fullcontact.person.email('foo@bar.com', 1, function (err, data) {
 });
 ```
 
+
+The `Person` API's now accept the `macromeasures` param as a boolean
```

Comment:
I would put this behind the webhook stuff just so that the arguments make more sense when reading for the first time.
It is also more likely people are looking for webhooks then the premium features I think